.github/workflows/organize-tags.yml

```diff
@@ -0,0 +1,63 @@
+name: Organize Tags
+
+on:
+  workflow_call:
+    inputs:
+      image:
+        required: true
+        type: string
+  workflow_dispatch:
+    inputs:
+      image:
```

Comment:
IIRC, a variable like this is called package somewhere else?

Reply:
https://github.com/vulsio/vuls-data-db/compare/f6cc7349043c37cb853ec9c226bc879826f76993..ae7fc852bddb21867358088c4b42e763cbfe2508